src/components/AccountLoginForm.jsx

```diff
@@ -8,7 +8,9 @@ import Field, { PasswordField, DropdownField, CheckboxField } from './Field'
 import ReactMarkdownWrapper from './ReactMarkdownWrapper'
 import FixedProgress from './FixedProgress'
 
-const AccountLoginForm = ({ t, isOAuth, oAuthTerminated, fields, error, dirty, submitting, forceEnabled, deleting, values, submit, onDelete, onCancel, connectorSlug, isSuccess, onAccountConfig, disableSuccessTimeout, isUnloading }) => {
+import AccountForceConnection from './AccountForceConnection'
```

Comment:
I think that `AccountLoginForm` is not the right place for a component like `AccountForceConnection`.
I suggest instead `AccountConnection` or better, `KonnectorSync`.